content/doc/book/pipeline/overview.adoc

```diff
@@ -1,4 +1,297 @@
 ---
-layout: refresh
-refresh_to_post_id: '/doc/book/pipeline/getting-started'
+layout: section
 ---
+:notitle:
+:description:
+:author:
+:email: jenkinsci-docs@googlegroups.com
+:sectanchors:
+:toc:
+:imagesdir: /doc/book/resources
+:hide-uri-scheme:
+
+
+= Pipeline Overview
+
+This section of the handbook will walk through all aspects of creating a Jenkins
+Pipeline beginning with very simple examples and progressing to more complex
+use-cases.
+
+Pipelines are a series of steps that allow you to orchestrate the work required
+to build, test and deploy applications. Pipelines are defined in a file called
+`Jenkinsfile` that is stored in the root of your project's source repository.
+Pipeline can be used to use almost any tool and work with any language. For example:
```

Comment:
Line 23 is awkward ("Pipeline can be used to use almost any tool"). Better: "Pipeline can accommodate almost any tool and work with almost any language."